### ISSUE TITLE
Data uncertainty?

### ISSUE THREAD

**Issue by ZhouCX117** [Li-Wanhua/POEs#5]
Thanks for your share and rep[ly. There is a new question. I am not good at distinguishing between data uncertainty and model uncertainty. Does POE models data uncertainty?

**Comment by Li-Wanhua** [comment]
Hi, the POE is used to model the data uncertainty. The difference between model uncertainty and data uncertainty is described in the paper "What Uncertainties Do We Need in Bayesian Deep Learning for Computer Vision?".  I believe that one important feature is that data uncertainty cannot be reduced even if more data were to be collected while the model uncertainty can. Hope this can help you.

**Comment by ZhouCX117** [comment]
Thanks for your reply! You are so nice~
Could I ask one more question?
Are there any other methods to measure model uncertainty besides Bayesian neural network, ensembles, and MC dropout?

**Comment by Li-Wanhua** [comment]
I believe that the most popular methods are the listed above methods. But, of course, you can always find some other methods including:
"Diverse Ensembles Improve Calibration",
"Hyperparameter Ensembles for Robustness and Uncertainty Quantification",
"Simple and Principled Uncertainty Estimation with Deterministic Deep Learning via Distance Awareness" and so on...

**Comment by ZhouCX117** [comment]
Thanks for your reply~ 
However, I notice a new question. How do you prove consistency when you use the rank-based method?  The inconsistency means the results show that it is higher than 20 but lower than 15.
